### PR TITLE
Added SQL Question

### PR DESCRIPTION
**Pull request by HetuKariya**
## Summary of changes
Added SQL Question #17 

## Checklist

- [✓] I have read and followed the [contribution guidelines](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CONTRIBUTING.md).
- [✓] I have read through the [Code of Conduct](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CODE_OF_CONDUCT.md) and agree to abide by the rules.
- [✓] This PR is for one of the available issues and is not a PR for an issue already assigned to someone else.
- [✓] My PR title has a short descriptive name so the maintainers can get an idea of what the PR is about.
- [✓] I have provided a summary of my changes.

closes #XXXXX

### CODE REVIEW

**Review by jdwilkin4 — COMMENTED** [review]
some of these questions have already been merged in.
please update your branch to reflect that.

File: src/data/sql-quiz.ts [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-    Question: "Which of the following is the most effective way to protect against SQL injection attacks?", 
-    Answer: "Using a modern SQL library that sanitizes SQL inputs",
-    Distractor1: "Using strong passwords for database access",
-    Distractor2: "Disabling certain database permissions",
-    Distractor3: "Encrypting database tables",
+    Question: "In terms of Big-O complexity, what is the time complexity for lookups in a binary tree index?", 
+    Answer: "O(log n)",
+    Distractor1: "O(n)",
+    Distractor2: "O(n log n)",
+    Distractor3: "O(1)",
```
Comment (jdwilkin4) [inline src/data/sql-quiz.ts:7]: this question should be added to the general cs quiz.
But also, you shouldn't override the first question of the SQL quiz.
Please fix this. Thank you

**Comment by jdwilkin4** [comment]
@HetuKariya 

It looks like you have some unresolved review comments.
Are you still interested in contributing?

**Comment by HetuKariya** [comment]
I added the first question again in the typescript file keeping the second question as it is just like you said um now what else is to be resolved?

**Comment by jdwilkin4** [comment]
@HetuKariya 

You need to move the sql question to the general CS file because that is where it would be more appropriate.

See my earlier comment here
Screenshot 2024-10-28 at 4 58 50 AM

**Review by jdwilkin4 — APPROVED** [review]

**Pull request merged**